lib/src/view/menu.dart

```diff
@@ -673,6 +673,16 @@ When selected, the length of each loopout is displayed next to it.'''
 // help menu
 
   help_menu() {
+    List<dynamic> older_versions_links = [for (var version in constants.scadnano_versions) DropdownItem(
```

Comment:
Does the React menu component let you make sub menus? Like this:

![image](https://user-images.githubusercontent.com/19274365/99409506-5030f500-28a6-11eb-9b67-a2b09bf70469.png)


That would be ideal here, to have a submenu named "Other versions", and then put dev and the older versions under it.

We may also want to start doing that with some of the clutter on other menus, for example the four ways to save to localStorage under File might be cleaner to have as a sub-menu, or grouping the various "Show/Hide" options in View under a single "Show/Hide" submenu.

Reply:
Resolved in https://github.com/UC-Davis-molecular-computing/scadnano/pull/510/commits/d19ac32b88656de64e26a42c8e96e27cd0ef15ee

Created issue #515 to address the cleanup issue